applications/GeoMechanicsApplication/custom_utilities/math_utilities.hpp

```diff
@@ -1236,5 +1236,8 @@ class StructuralMechanicsMathUtilities
 
 private:
 };// class StructuralMechanicsMathUtilities
+
+KRATOS_API(GEO_MECHANICS_APPLICATION) double DegreesToRadians(double AngleInDegrees);
```

Comment:
You're declaring the function at the global Kratos namespace. Though this is technically possible, in Kratos we always prefer the `static` but inside a class approach. As suggested by @philbucher, I'd place this within the Kratos core math utils.

Reply:
Thanks, I will move the function to the Kratos core math utils as suggested by @philbucher and you. However, what is the rationale behind using a static member function rather than a nonmember function? If you're trying to limit its scope (which is a fair point), I feel that using a nested namespace is a simpler and cleaner way to achieve just that.
I will follow your suggestion anyway, but I'm just asking you to think about this one and to consider whether we may want to change this policy. Thanks.

Reply:
Yes, my point is to basically limit its scope. About how to achieve it, i.e. the class vs namespace, this is a long term discussion in Kratos. At some point we tried to provide some guidelines & conclusions in #7434.